### DISCUSSION TITLE
Error while running pipeline in kubeflow Editor

### DISCUSSION THREAD

**Discussion by vinaykd25** [elyra-ai/elyra#3077] [Pipeline Editor/ Kubeflow Pipelines + Apache Airflow]
We are using the notebook image build on top of the **quay.io/elyra/kf-notebook:3.14.1** as a base image. We rae trying to run simple pipeline example but not able to do so. Our kubeflow running on K8s and ebaled with dex ldap authentication. We have provided all information in runtime 
![image](https://user-images.githubusercontent.com/43445324/212301394-8c874f63-86ef-44bc-86e8-e5ecdcdf0217.png)
I am able to run the pipeline in generic Pipeline Editor, but gettig error unexpected end of JSON input in kubeflow pipeline editor
![image](https://user-images.githubusercontent.com/43445324/212302701-38892ce9-e21a-4141-8c9c-79a62fcd9f4a.png)

Please help me out here

**Comment by kevin-bates** [comment]
Hi @vinaykd25 - from your gitter post you mention that you're unable to see anything related to the logs in the JupyterLab console (terminal) or wherever that output may have been redirected.
> Not able to get logs related to this while checking the jupyter lab logs

Are you saying that _nothing_ is logged to the console the moment you submit the KF pipeline to run (or export)?  At a minimum, I would expect the typical REST logging (probably for `POST` against `/elyra/pipeline/schedule` in this case) to be logged.  If that's the case, then it might be helpful to check the browser log.

Can you please provide what server-side console output _is_ produced (if any) and perhaps check (and provide) any output in your browser log (i.e., java-script console, etc.)?

(I believe this should also be transferred to the issues list, but let's see if this can be resolved in a step or two.)

**Reply by vinaykd25** [reply]
Hi @kevin-bates Thanks for your response here are my pod logs server side
<img width="657" alt="image" src="https://user-images.githubusercontent.com/43445324/212471282-5433b534-fb1c-47e5-867a-e8ab5431fa0d.png">

<img width="643" alt="image" src="https://user-images.githubusercontent.com/43445324/212471175-f9d894cf-4478-468b-aabf-6b2603c7d4ed.png">

**Reply by ptitzler** [reply]
Just as an fyi, the second error (`AttributeError: 'HTTPErrorMixIn' object has no attribute ...`) is unrelated and will be fixed by https://github.com/elyra-ai/elyra/pull/3080 in 3.14.2.

**Comment by kevin-bates** [comment]
This looks like an authentication-related configuration issue.  Unfortunately, I'm not the right person to troubleshoot this but someone should be able to get you going.  I think this discussion is better served as an issue and will transfer it to that list.

**Reply by vinaykd25** [reply]
I already have cert. I wanted to know where I can apply these cert?

**Reply by ptitzler** [reply]
Elyra uses the `requests` package under the covers. This package evaluates the `REQUESTS_CA_BUNDLE` environment variable. If you set this variable to your certificate file location in the environment where JupyterLab/Elyra is running then this should work. See  https://requests.readthedocs.io/en/latest/user/advanced/#ssl-cert-verification for details. (The runtime configuration currently does not support specification of a custom certificate file location.)

**Reply by vinaykd25** [reply]
Thanks @ptitzler for response. Issue is resolved. I have to apply the our ssl cert at **/opt/conda/lib/python3.8/site-packages/certifi/** 
After that issue was no more I am able to see the pipeline in Kubeflow now. Below is what I added in my docker image
![image](https://user-images.githubusercontent.com/43445324/213384886-07a0616a-efd2-4001-a439-68f0ca5a4711.png)

**Reply by ptitzler** [reply]
Glad to hear!
